02-Logging/02-Writing-to-a-File.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<p>Same question as previously, I'm not sure what the benefit of returning these quantities from <code>logger.add</code>  here. Not a question about the tutorial so much as a question about HOOMD.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>